src/harry-potter-app/components/card/Card.js

```diff
@@ -64,7 +64,7 @@ function Card({
           onEmojiButtonClick={(newEmoji) => handleEmojiButtonClick(newEmoji)}
         />
 
-        <CardHeader>
+        <CardHeader role="heading">
```

Comment:
I believe it should work without setting the role, since h2 should automatically get the role heading. Not sure, but if so, you can skip setting it here.

Reply:
Yes, you're right. I removed it.